LICENSE.rtf

```diff
@@ -1,8 +1,8 @@
 {\rtf1\ansi\ansicpg1252\deff0\nouicompat{\fonttbl{\f0\fnil\fcharset0 Courier New;}{\f1\fnil\fcharset77 Courier New;}}
 {\*\generator Riched20 6.3.9600}\viewkind4\uc1 
 \pard\sa200\sl276\slmult1\f0\fs18\lang12 MuseScore, free and open source music notation software.\par
-Copyright (C) 1999-2020 MuseScore BVBA and others.\par
-This program is free software; you can redistribute it and/or modify it under the terms of the GNU General Public License version 2 as published by the Free Software Foundation and reproduce below with the following exception:\par
+Copyright (C) 1999-2021 MuseScore BVBA and others.\par
```

Comment:
Or should we make it 2022 already?

Reply:
Is that a 'yes'? Or wait until we're really in 2022 (there are more places to make that change, I believe)

Reply:
Guess it really depends on when an Alpha gets released, for Beta, RC and Final Release we'd certainly need to change it anyhow

Reply:
@igorkorsukov cherry-pick as you see fit...

Reply:
>there are more places to make that change

Anyway, one of those places would be src/main/CMakeLists.txt, line 78, for the MACOSX_BUNDLE_COPYRIGHT.

For the rest, I can't really think of other places where it would be exposed to the user. However, it is of course at the top of each source file, but that's probably better to solve in a separate "happy new (copyright) year" pull request.

Reply:
Added